### ISSUE TITLE
Usermin Configuration menu item cannot be moved from unused modules 

### ISSUE THREAD

**Issue by durangod** [webmin/webmin#428]
Hi, i hope this is the right place to post this, if not i sincerely apologize.  I installed Usermin with my Webmin and Virtualmin install and the Usermin modules will not move from the unused modules category.   I wanted to move it out of there but it wont go. 

Any ideas, thanks so much :0

**Comment by jcameron** [comment]
Did you try clicking the Refresh Modules button / link on the left menu?

**Comment by durangod** [comment]
yes sir i just tried that and its still sitting in unused did not move.

**Comment by durangod** [comment]
update OK i got it to move, after i did the refresh modules just by curiosity i decided to move it somplace else and it worked, not sure why this time, but it hade something to do with doing the refresh that unlocked it a guess so i could move it. 

all better now thanks :)
